### DISCUSSION TITLE
Note: Deno in js resembles fsx with nuget quite a lot

### DISCUSSION THREAD

**Discussion by jkone27** [dotnet/fsharp#15125] [General]
Thought maybe is interesting for folks working on dotnet fsi and .fsx scripts with nuget :) 

seems deno in js ecosystem is taking a similar approach

https://deno.land/manual@v1.28.1/node/npm_specifiers

**Comment by eriawan** [comment]
@jkone27 

This is more like into discussion instead of a true/real issue.
I suggest to move this into discussion.

**Reply by jkone27** [reply]
yes, it was more of a note/suggestion/interesting material maybe, ~ "nuget: x" that we have in fsx
